instructionAPI/src/InstructionDecoder-amdgpu-vega.C

```diff
@@ -154,15 +155,10 @@ namespace Dyninst {
 			return makeAddExpression(makePCExpr(), Immediate::makeImmediate(Result(u64, unsign_extend64(3, 4))), u64);
 		}
 
-		bool InstructionDecoder_amdgpu_vega::pre_process_checks(const amdgpu_insn_entry &entry) {
-			bool ret = false;
-			return ret;
-		}
 
 		bool InstructionDecoder_amdgpu_vega::decodeOperands(const Instruction *insn_to_complete) {
 			assert(0 && "decodeOperands deprecated for amdgpu");
-
-			return true;
+			return insn_to_complete != NULL;
```

Comment:
Same as above.

Reply:
Revert back to return true, and the parameter name is removed.